### ISSUE TITLE
Support autocomplete when searching for properties

### ISSUE THREAD

**Issue by jialin-wu-02** [aimhubio/aimui#35]
**Problem**
Search autocomplete would allow faster and more seamless search experience for the AI engineers.
Autocomplete could suggest the following properties:
- search keywords (`metric`, `param`, `tag`, `tf_scalar` and others to come)
- corresponding to the search keyword, suggest appropriate values based on the history of experiments. For instance suggest metrics on `metric:` to fill in the corresponding metric
- corresponding values of the search: for instance in case of `param:learning_rate=` suggest potential values from the history of corresponding experiments

**Solution**
Add autocomplete to search to make the experience more seamless. The details are described in the Problem section.
